docs/howto/custom-shell.txt

```diff
@@ -0,0 +1,42 @@
+============================================= 
+How to customize the :djadmin:`shell` command 
```

Comment:
does a link in a title work well? I would guess not. Did you build the docs locally?

Regardless, let’s use plain text in the title:

```suggestion
How to customize the ``shell`` command 
```

The link can live only in the body text below.